src/components/visualizations/chart/highcharts/getOptionalStackingConfiguration.ts

```diff
@@ -141,7 +153,11 @@ function hasNegativeValues(series: ISeriesItem[] = [], index: number) {
     return isNegativeValueIncluded(matchedSeries);
 }
 
-export function getYAxisConfiguration(chartOptions: IChartOptions, config: any, chartConfig: IChartConfig) {
+export function getYAxisConfiguration(
+    chartOptions: IChartOptions,
+    config: any,
+    chartConfig: IChartConfig,
+): { yAxis?: any[] } {
```

Comment:
this is not usual way how to specify return type... is it possible to specify an interface? that we could figure out with what we are working ??